validators/types_unix.go

```diff
@@ -67,9 +68,12 @@ var DefaultSysSpec = SysSpec{
 		// The hugetlb cgroup is optional since some kernels are compiled without support for huge pages
 		// and therefore lacks corresponding hugetlb cgroup
 		"hugetlb",
+		// The blkio cgroup is optional since some kernels are compiled without support for block I/O throttling.
+		// Containerd and cri-o will use blkio to track disk I/O and throttling in both cgroup v1 and v2.
+		"blkio",
```

Comment:
If the CRs are going to also track v1, we should add it under v1 as optional here ad well?

Reply:
Yes, I add it to both v1 and v2.

- CgroupsOptional is for v1
- CgroupsV2Optional is for v2

like hugetlb, I add comments to explain in v1 only.

Reply:
ok, SGTM.